src/utils/public-ip.ts

```diff
@@ -65,20 +67,33 @@ async function pokeStun(): Promise<string> {
 }
 
 export async function getPublicNetworkInfo(): Promise<PublicNetwork> {
+  const { flags } = getContext()
+  const isSymonMode = isSymonModeFrom(flags)
+
   const publicIp = await pokeStun()
   const response = await sendHttpRequest({
     url: `http://ip-api.com/json/${publicIp}`,
   })
   const { country, city, isp } = response.data
 
-  publicNetworkInfo = {
-    country,
-    city,
-    hostname: hostname(),
-    isp,
-    privateIp: getIp(),
-    publicIp,
-  }
+  publicNetworkInfo =
+    flags.verbose || isSymonMode
```

Comment:
Don't add the conditional statement here. Put it in the `getPublicNetworkInfo` caller to make the function easier to read and test.